sensing/autoware_pointcloud_preprocessor/src/distortion_corrector/distortion_corrector.cpp

```diff
@@ -187,15 +188,98 @@ bool DistortionCorrector<T>::isInputValid(sensor_msgs::msg::PointCloud2 & pointc
   return true;
 }
 
+template <class T>
+bool DistortionCorrector<T>::azimuthConversionExists(sensor_msgs::msg::PointCloud2 & pointcloud)
```

Comment:
This naming is a bit misleading: it suggests that the existence of a conversion is checked and returned, but in reality the conversion is computed **and causes a side effect** (storing values in `a_` and `b_`).

This function would be better if renamed to something like `tryComputeAndSetAzimuthConversion`.
Even better, it would not set `a_` and `b_` directly but return a `std::optional<AngleTransform>` and be called `tryComputeAzimuthConversion`.

Reply:
Thanks! fixed in https://github.com/autowarefoundation/autoware.universe/pull/8380/commits/8a24a79457c1204a21b0a398d90a4ad5185ea82e